tests/Fractions.Tests/FractionSpecs/Sqrt/Method_Sqrt.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using FluentAssertions;
+using NUnit.Framework;
+using Tests.Fractions;
+
+namespace Fractions.Tests.FractionSpecs.Sqrt {
+
+    [TestFixture]
+    public class If_the_Sqrt_function_is_called : Spec {
+        [Test]
+        public void Sqrt_Of_1_Shall_Be_1() {
+            Fraction.One.Sqrt()
+                .Should().Be(1);
+        }
+
+        [Test]
+        public void Sqrt_Of_MinusOne_Shall_Be_Fail() {
+
+            Action act = () => Fraction.MinusOne.Sqrt();
+
+            act.Should().Throw<OverflowException>()
+                        .WithMessage("Cannot calculate square root from a negative number");
+        }
+
+        
+        private bool SqrtCompareToDouble(double testValue) {
+
+            var fractionSqrt = Fraction.FromDouble(testValue).Sqrt();
+            var doubleSqrt = Math.Sqrt(testValue);
+
+            return fractionSqrt.ToDouble() == doubleSqrt;            
+        }
+
+        [Test]
+        public void Sqrt_Of_values() {
+
+            SqrtCompareToDouble(0.001).Should().BeTrue();
+            SqrtCompareToDouble(5).Should().BeTrue();  
+            SqrtCompareToDouble(4.54854751).Should().BeTrue();
+            SqrtCompareToDouble(9999999855487).Should().BeTrue();
+            SqrtCompareToDouble(99999998554865557).Should().BeTrue();
+            SqrtCompareToDouble(Math.PI).Should().BeTrue();
+
+        }
+
+
+
+
+
```

Comment:
Yeah, the tests... *Fractions* started as private project. Nearly every single line was written in German language 🥲. I am sorry for not writing proper tests in the first place. However, the use of TestCases makes more verbose tests:

![image](https://user-images.githubusercontent.com/3825484/154846261-dcba18b9-14ba-42af-a9d4-5b8245118869.png)

**CAUTION** SUGGESTION REQUIRES A NEW USING `using System.Collections.Generic;` 

```suggestion
        private static IEnumerable<TestCaseData> TestCases {
            get {
                yield return new TestCaseData(0.001);
                yield return new TestCaseData(5);
                yield return new TestCaseData(4.54854751);
                yield return new TestCaseData(9999999855487);
                yield return new TestCaseData(99999998554865557);
                yield return new TestCaseData(Math.PI);
            }
        }

        [Test,TestCaseSource(nameof(TestCases))]
        public void Should_the_result_be_equal_to_Math_Sqrt(double value) {
            var expected = Math.Sqrt(value);
            var actual = Fraction.FromDouble(value).Sqrt();
            actual.ToDouble().Should().Be(expected);
        }```